apps/omg_eth/test/omg_eth/ethereum_client_monitor_test.exs

```diff
@@ -12,31 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-defmodule OMG.EthereumClientMonitorTest do
+defmodule OMG.Eth.EthereumClientMonitorTest do
```

Comment:
heeey, since you're doing some more profound changes to this already, maybe you could look into the speed of these tests?

```
  mix test test/omg --slowest
    Top 1 slowest (4.6s), 76.0% of total time:

      * test that we don't overflow the message queue with timers when Eth client needs time to respond (4696.1ms) (OMG.EthereumClientMonitorTest)
```

(on my "close to `master`" branch)

It would be awesome if `mix test` run faster, I end up skipping this test manually when I need to use `mix test` a lot.